versions/v6/templates/story/related-stories.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$related_stories = related_stories_get_stories( $post->ID );
+
+if ( $related_stories ) :
+?>
+<aside>
+	<h2>Related Stories</h2>
+	<?php foreach( $related_stories as $story ) : ?>
+		<article class="d-block mb-4">
+			<img alt="" class="img-fluid" src="<?php echo $story['thumbnail']; ?>">
```

Comment:
Adding width and height attributes on each `<img>` would provide a small performance benefit, if we can grab them and pass them in to `$story`

Reply:
This isn't a problem for the local Pegasus stories, but may be an issue for the Today stories. I will look into it while we're working on the styling for this.